src/IntercomConversations.php

```diff
@@ -35,6 +35,20 @@ public function getConversation($id, $options = [])
         return $this->client->get($path, $options);
     }
 
+    /**
+     * Returns list of Conversations that match search query.
+     *
+     * @see     https://developers.intercom.com/reference#search-for-conversations
+     * @param   array $options
+     * @return  stdClass
+     * @throws  Exception
+     */
+    public function searchConversations(array $options)
```

Comment:
I would also rename this to `search`. I see the rest of the methods include the word conversations, but to be honest I would prefer to stick to `search` and keep the goal to rename the rest in a future version